eventmesh-server-go/config/common.go

```diff
@@ -0,0 +1,24 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+type Common struct {
```

Comment:
It's better to rename Common to CommonConf or CommonConfig?

Reply:
As its own package name is `config`, it can be imported through `config.Common` when it is referenced externally. It can be known that it is a common configuration. If it is changed to `CommonConfig`, the external reference becomes `config.CommonConfig`. There are two configs in package name and structure name, which is generally not recommended in Go.